plugin/advancedsubscription/ajax/advsub.ajax.php

```diff
@@ -0,0 +1,248 @@
+<?php
+/* For licensing terms, see /license.txt */
+/**
+ * Script to receipt request to subscribe and confirmation action to queue
+ * @author Daniel Alejandro Barreto Alva <daniel.barreto@beeznest.com>
+ * @package chamilo.plugin.advancedsubscription
+ */
+
+/**
+ * Init
+ */
+require_once __DIR__ . '/../config.php';
+
+$plugin = AdvancedSubscriptionPlugin::create();
+// Get validation hash
+$hash = Security::remove_XSS($_REQUEST['v']);
+// Get data from request (GET or POST)
```

Comment:
Don't use this kind of variable shortcuts. Use complete words.
For example:  $data['action']  instead of $data['a']